backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -222,13 +223,16 @@ def _generate_privilege_record(
         jurisdiction_postal_abbreviation: str,
```

Comment:
Accessibility: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.3:1. Recommendation: change text colour to #2d2d2d.


<img width="321" alt="image" src="https://github.com/user-attachments/assets/67d230e3-e410-46a1-91b8-37f0bc12e7f3">


@jsandoval81